src/encoded/static/components/facets.js

```diff
@@ -7,6 +7,14 @@ import { initializedChromatinObject } from './chromatin_view';
 // Number of terms to show, the rest will be viewable on scroll
 const displayedTermsCount = 14;
 
+// Add method to filter objects based on value
+Object.filter = (obj, predicate) =>
+    Object.keys(obj)
+        .filter(key => predicate(obj[key]))
+        .reduce((res, key) => Object.assign(res, { [key]: obj[key] }), {});
```

Comment:
Interesting. I didn’t know extending Object with prototype was frowned upon, though the page they linked to didn’t really say that, or maybe I misunderstood (not that I would extend the prototype of Object in my own code). Anyway, I don’t see the advantage of this over just a regular function to do the filtering, as the poster also said, “Instead, provide your feature as stand-alone function, or if you really want to extend a global, provide it as utility function on Object.”